docs/network-flow-visibility.md

```diff
@@ -610,3 +613,58 @@ and other Theia features, please refer to the
 **Starting with Antrea v1.7, support for the ELK Flow Collector has been removed.**
 Please consider using the [Grafana Flow Collector](#grafana-flow-collector-migrated)
 instead, which is actively maintained.
+
+## Layer 7 network flow export
+
+In addition to layer 4 network visibility, Antrea adds layer 7 network flow
+export.
+
+### Prerequisites
+
+To achieve L7 (Layer 7) network flow export, the `L7FlowExport` feature gate
+must be enabled.
+
+### Usage
+
+To export layer 7 flows of a Pod or a Namespace, user can annotate Pods or
+Namespaces with the annotation key `visibility.antrea.io/enable-l7` and set the
+value to indicate the traffic flow direction, which can be `ingress`, `egress`
+or `both`.
+
+For example, to enable L7 flow export in the ingress direction on
+Pod test-pod in the default Namespace, you can use:
+
+```bash
+kubectl annotate pod test-pod visibility.antrea.io/enable-l7=ingress
+```
+
+Based on the annotation, Flow Exporter will export the L7 flow data to
+the Flow Aggregator using the fields `appProtocolName` and `httpVals`.
```

Comment:
The receiver can be other IPFix collectors right.

Reply:
Yes

Reply:
Then let us say "to the Flow Aggregator or configured IPFix collector".

And probably we should say "Flow Exporter in Antrea Agent will export ..."